app/controllers/catalog_controller.rb

```diff
@@ -48,7 +48,7 @@ class CatalogController < ApplicationController
     config.add_index_field 'ending_date_ssim', label: 'Ending date'
     config.add_index_field 'dated_mss_ssim', label: 'Dated Mss'
     config.add_index_field 'annotation_text_tesim', label: 'Annotation text'
-    config.add_index_field 'annotation_tags_ssim', label: 'Annotation tags', link_to_facet: true
+    config.add_index_field 'annotation_tags_facet_ssim', label: 'Annotation tags', link_to_facet: true
```

Comment:
Hmm, I'm a little confused.  Are we render the field that is formatted (`Animali:Agnelli`) in the search result UI?

Also. should this change the [hierarchical facet config](https://github.com/sul-dlss/vatican_exhibits/pull/158/files#diff-956e46b33fb5307990c4e1a4b5fd86ccL133) to match this new `:` delimited `annotation_tags_facet_ssim` field?

Perhaps Blacklight is doing something more clever here than I thought.

Reply:
Nope, you're right. I changed the wrong occurance of `annotation_tags_ssim`